### PR TITLE
Workspace

### PR DESCRIPTION
**Pull request by INVEEk**
Exercise from 1 to 7

### CODE REVIEW

File: exercise_17.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    }
+    if (operation === "-") {
+        return value1 - value2
+    }
+    if (operation === "*") {
+        return value1 * value2
+    }
+    if (operation === "/") {
+        return value1 / value2
+    }
```
Comment (WojciechBaczynski) [inline exercise_17.js]: here also value2 cannot be 0!

File: exercise_20.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+name + " does not play banjo"
+Names given are always valid strings.
+ */
+
+function areYouPlayingBanjo(name) {
+    if (name[0] === "r" || name[0] === "R") {
+        return name +  ` plays banjo`;
+    } else {
+        return name + ` does not play banjo`;
+    }
```
Comment (WojciechBaczynski) [inline exercise_20.js]: typically when we use backtics then we write it like:
```
return `${name} plays banjo`;
```
and when we do not then:
```
return name + " plays banjo";
or
return name + ' plays banjo';
or
return name + ' ' + 'plays banjo';
or
return name + " " + "plays banjo";
```

File: exercise_17.js (hunk 1 above)
Comment (INVEEk) [reply exercise_17.js]: Fixed and pushed to github

File: exercise_20.js (hunk 2 above)
Comment (INVEEk) [reply exercise_20.js]: Fixed and pushed to github

File: exercise_17.js (hunk 1 above)
Comment (WojciechBaczynski) [reply exercise_17.js]: I cannot see a difference

File: exercise_17.js
Comment (INVEEk) [inline exercise_17.js:1]: Exercise 17 fixed. Alert shown once user fill operations with 0 as a value2

**Pull request merged**